client/src/components/DateTime/styles.js

```diff
@@ -76,10 +76,14 @@ const styles = StyleSheet.create({
     color: Colors.bdMain,
   },
   timeSelectContainer: {
+    width: '100%',
     flexDirection: 'row',
-    justifyContent: 'space-around',
     flexWrap: 'wrap',
   },
+  timeSelectChild: {
+    width: '48%',
+    margin: '1%',
```

Comment:
i'm sure this will bite us at some point - I feel like this kinda stuff never works in CSS so it probably doesn't work here either. but it'll do for now.

Reply:
I couldn't find an effective way other than this to lay out the timeSegment tiles - given the length of titles are different. We can revisit in the future.